trace/header.go

```diff
@@ -1,34 +1,60 @@
 package trace
 
 import (
+	"log"
 	"net/http"
-
-	"github.com/twinj/uuid"
 )
 
 const (
-	HeaderId = "x-request-id"
-
-	HeaderParentId = "x-parent-id"
+	// HeaderSpanID is the header name in requests for the SpanID.
+	HeaderSpanID = "Trace-Span"
+	// HeaderTraceID is the header name in requests for the TraceID.
+	HeaderTraceID = "Trace-Trace"
 )
 
-func GetId(r *http.Request) string {
-	return r.Header.Get(HeaderId)
+// Generates a new SpanID from a request. If there is no span information in the request, a root SpanID will be generated.
```

Comment:
The function/method documentation usually starts with the name of the function/method. 

> Notice this comment is a complete sentence that begins with the name of the element it describes. This important convention allows us to generate documentation in a variety of formats, from plain text to HTML to UNIX man pages, and makes it read better when tools truncate it for brevity, such as when they extract the first line or sentence.

Source: http://blog.golang.org/godoc-documenting-go-code